qdrant_client/local/local_collection.py

```diff
@@ -588,7 +634,28 @@ def _preprocess_recommend(
         using: Optional[str] = None,
         lookup_from_collection: Optional["LocalCollection"] = None,
         lookup_from_vector_name: Optional[str] = None,
-    ) -> Tuple[List[List[float]], List[List[float]], types.Filter]:
+    ) -> Tuple[
+        List[List[float]],
+        List[List[float]],
+        List[models.SparseVector],
+        List[models.SparseVector],
+        types.Filter,
+    ]:
+        def split_examples(
```

Comment:
I had to read a few times to understand what this was doing, maybe:
```suggestion
        def examples_into_vectors(
```
?